File: Code/cpp/is_digit.cpp
#include<iostream>
using namespace std;
int main()
{
    char input;

    cout << "Enter Input: ";
    cin >> input;

    if (isdigit(input) != 0)
        cout << "Is Digit";
    else
        cout << "Is not Digit";
    return 0;
}

File: Code/java/is_digit.java
import java.util.Scanner;

class Digit
{
    public static void main(String[] args)
    {
        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter Input: ");
        try
        {
            Float.parseFloat(scanner.next());
            System.out.println("Is Digit");
        }
        catch(NumberFormatException e)
            {System.out.println("Is not Digit");}

        scanner.close();
    }
}